app/models/release_package.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ReleasePackage
+  include ActiveModel::Model
```

Comment:
Let's turn this into an active record. We can replace the product's `distribution_engine` with a package `engine`, and allow releases to be associated to a specific package, allowing multiple distributions under the same product.

Reply:
```rb
class ReleasePackage
  attribute :product_id, :uuid
  attribute :keywords, :string, array: true
  attribute :author, :string
  attribute :license, :string
  attribute :engine, :string
end
```